### DISCUSSION TITLE
740

### DISCUSSION THREAD

**Discussion by giscus** [jojoldu/blog-comments#922] [Comments]
# 740

대학교를 다닐때 학교 공부는 하지 않고, 도서관에서 자기계발서를 한참 읽던 시기가 있었다. 그때 읽었던 책 중에 하나가 젊은 구글러가 세상에 던지는 열정력이라는 책이었는데, 200X년에 구글코리아에 입사하신 김태원님이 쓰신 책이다. 각 챕터마다 김태원님의 생각이 적힌 책인데, 그 중에서 "두근두근" 챕터에는 수잔 베이츠의 "사람을 움직이는 리더의 대화법" 의 내용이 담겨 있었다. 교회는 이미 사람들로 꽉 차 있었고, 사진기자와 나는 교회 뒷문에 진을 칠 계획으로 언덕에 올랐다. 그때 나는 마침 따르는 기자들 없이 경호원 두 명과 걸어가는 지미 카터를 보았다. 그들은 막 교회로 들어가려던 참이었고, 내 심장은 빠르게 뛰었다. 이런 기회가 오리라고 예상하지 못했던 나는 아무런 준비도 되어 있지 않은 상황이었..

https://jojoldu.tistory.com/740

**Comment by ryusimyeong** [comment]
10초만 쪽팔림을 감수할 용기가 있다면 인생이 더 멋져질 거라는 말을 들었는데 비슷한 내용인 거 같네요. 잘 봤습니다.

**Comment by AppleCEO** [comment]
떨린다, 긴장된다 -> 두근거린다
평소에 위처럼 치환하려고 하는데 훨씬 긍정적인 표현인 것 같아요~

**Comment by Hoo93** [comment]
항상 느끼는 것이지만 감동 받은 내용을 길지 않게, 담백하게 잘 풀어내시는 것 같아요.
저는 여러차례 면접을 봤지만 긴장되는 순간을 이겨내기 쉽지 않았는데 생각해 보면 두근두근하는 모든 순간이 기회였네요.
좋은 글 감사합니다 ! 앞으로는 두근거림을 즐겨보겠습니다 !

**Comment by GangEunzzang** [comment]
머리가 띵하네요 ! 
두려움을 즐길수 있는 인생을 살아보겠습니다 !!
